share/spice/dota2/content.handlebars

```diff
@@ -0,0 +1,149 @@
+{{#if isItem}}
+    <div style="float: left; width: 100px;">
+        <img src="{{image}}" title="{{name}}"/>
+    </div>
+    <div style="float: right; width: 500px;">
+        <h1 class="zci__header">
+            {{name}}
+        </h1>
+        <hr />
+        <div class="twenty text--secondary">
+            Cost
+        </div>
+        <div class="eighty">
+            <img src="http://cdn.dota2.com/apps/dota2/images/tooltips/gold.png" title="Gold"/> {{cost}}
```

Comment:
You'll need to use our `imageProxy` helper here so the client's IP isn't leaked to dota2.com: https://github.com/duckduckgo/duckduckgo-template-helpers/blob/ee1713f5218c18a880a133583415cb09d8088a1b/template_helpers.js#L336